README.md

```diff
@@ -19,6 +19,8 @@ If you have a max of 40 MB/second, 2 producers, and one of them is producing mes
 
 ## Using with Strimzi
 
+**Warning: this version is not included in Strimzi yet, see tag [0.2.0](https://github.com/strimzi/kafka-quotas-plugin/tree/0.2.0)** for current Strimzi-compatible docs.
```

Comment:
I wasn't sure of the right thing to do here. We don't want to mislead people arriving at https://github.com/strimzi/kafka-quotas-plugin.

Maybe we should remove the `#Using with Strimzi` section until it's integrated with Strimzi, leaving just the warning.

Reply:
I think the warning is good. Bu the actual configuration will be probably different at the end. So it might make sense to remove it. But it might make sense to have some section with pure Kafka configuration maybe?

Reply:
(I.e. keep it as you have it there already I guess)

Reply:
done, I've removed the strimzi usage details